hack/whitespace.sh

```diff
@@ -3,11 +3,11 @@
 set -e
 
 function fix() {
-    git ls-files -- ':!vendor/' | xargs sed --follow-symlinks -i 's/[[:space:]]*$//'
+    git ls-files -- ':!vendor/' ':!tools/vendor' | xargs sed --follow-symlinks -i 's/[[:space:]]*$//'
```

Comment:
nit: in order to exclude /vendor/ folders that might exists in the future
`git ls-files | grep -vE '^vendor/|/vendor/'`
or maybe also
`git ls-files -- ':!vendor' ':!**/vendor/**'`
but just nice to have please, it can wait for once we have new vendor folders